CMakeLists.txt

```diff
@@ -40,33 +55,57 @@ target_include_directories(ghexlib INTERFACE
 target_link_libraries(ghexlib INTERFACE GridTools::gridtools MPI::MPI_CXX)
```

Comment:
here you should link against `UCP::libucp` or `PMIx::libpmix` if this is enabled